scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskDataVariable.java

```diff
@@ -32,8 +25,18 @@
 
     private TaskData taskData;
 
+    static TaskDataVariable create(String variableName, TaskVariable taskVariable, TaskData taskData) {
+        TaskDataVariable taskDataVariable = new TaskDataVariable();
+        taskDataVariable.setName(variableName);
+        taskDataVariable.setValue(taskVariable.getValue());
+        taskDataVariable.setModel(taskVariable.getModel());
+        taskDataVariable.setJobInherited(taskVariable.isJobInherited());
+        taskDataVariable.setTaskData(taskData);
+        return taskDataVariable;
+    }
+
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue
```

Comment:
I had some issue related to the index, I simply reused a configuration which worked on another class. I did not invest more in understanding the problem.